src/pages/Account/Profile.tsx

```diff
@@ -61,57 +65,98 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
         </div>
-        <a
-          className="profile-picture-button Profile__summit-snap"
-          href="https://virtual.githubphotobooth.com/virtual/capture/gr99n"
-          {...externalUrlAdditionalProps}
-        >
-          Take a Summit snap
-        </a>
         <form onSubmit={handleSubmit(onSubmit)} className="form">
           <div className="input-group profile-form">
             {/* @debt - this input is hidden because for github's event they fetch the names from an OKTA, and for the demo we set the default name to Mona */}
             <input
               name="partyName"
               className="input-block input-centered"
               placeholder="Your display name"
-              defaultValue="Mona"
-              type="hidden"
+              defaultValue={githubHandle}
               ref={register({
                 required: true,
                 maxLength: DISPLAY_NAME_MAX_CHAR_COUNT,
               })}
               autoComplete="off"
             />
-            {/* <span className="input-info">
+            <p className="input-info">
               This is your display name (max {DISPLAY_NAME_MAX_CHAR_COUNT}{" "}
               characters)
-            </span> */}
-            {/* {errors.partyName && errors.partyName.type === "required" && (
+            </p>
+            {errors.partyName && errors.partyName.type === "required" && (
               <span className="input-error">Display name is required</span>
             )}
-            {errors.partyName &&* errors.partyName.type === "maxLength" && (
+            {errors.partyName && errors.partyName.type === "maxLength" && (
               <span className="input-error">
                 Display name must be {DISPLAY_NAME_MAX_CHAR_COUNT} characters or
                 less
               </span>
-            )} */}
+            )}
+            {/* TODO: Proper text, namings */}
+            <input
+              name="realName"
+              className="input-block input-centered"
+              placeholder="Your real name"
+              defaultValue={realName}
+              ref={register()}
+              autoComplete="off"
+            />
+            <p className="input-info">This is your real name</p>
+            <input
+              name="companyTitle"
+              className="input-block input-centered"
+              placeholder="Your title"
+              defaultValue={profilePrefilledData?.companyTitle}
+              ref={register()}
+              autoComplete="off"
+            />
+            <p className="input-info">This is your title</p>
+            <input
+              name="companyDepartment"
+              className="input-block input-centered"
+              placeholder="Your department"
+              defaultValue={profilePrefilledData?.companyDepartment}
+              ref={register()}
+              autoComplete="off"
+            />
+            <p className="input-info">This is your department</p>
```

Comment:
Will each of these need to have error display added to them?

Reply:
I believe these also may need/want to be added to `EditProfileForm`

Reply:
1) They are optional + no restrictions since they will be fetched from their Okta.
2) as per https://github.com/sparkletown/sparkle/pull/1667#discussion_r659621274 will work on this now, but it can be reviewed without that

Reply:
Using this thread to track the progress
https://github.com/sparkletown/sparkle/pull/1667#discussion_r659473073